### DISCUSSION TITLE
Octopus Mini IHD E-Paper

### DISCUSSION THREAD

**Discussion by anothermort** [BottlecapDave/HomeAssistant-OctopusEnergy#383] [Show and tell]
Have been using the Octopus Energy on Home Assistant for sometime and think it is very handy.

Was originally on ECO 7 have recently moved to Agile (less than a month).

Have been lucky enough to have an Octopus Mini for a week or so.

Realising that the IHD display connected to the meter is now redundant, as regards to pricing. Wanted something more convenient than jumping on the laptop or phone to check HA for price etc..

![octo_mini_ihd](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/assets/13786570/0dd72064-48d1-464f-a4fe-60576f0f8ac6)

Display uses an [ESP32 £7 from Amazon](https://www.amazon.co.uk/dp/B071JR9WS9/?coliid=I22Y46Q1TSMUTG&colid=4TI88AT1TSQY&psc=1&ref_=list_c_wl_lv_ov_lig_dp_it) programmed  via ESPhome
With a [Waveshare 2.9" E Paper screen £25.99 from Amazon ](https://www.amazon.co.uk/dp/B071LGVVL1?psc=1&ref=ppx_yo2ov_dt_b_product_details)

Currently displays Battery SOC % from Victron System, Time, Elec and Gas use so far today in kWh and Elec and Gas Cost so far today including standing charge. 

Could not find the costs to start off with but after help from Dave realised I was looking at the prices with standing charge. Thank you again Dave.

The data is updated every 30 seconds from HA.

Currently try to get a splash screen to display on startup.
![octomini ihd](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/assets/13786570/9ee344f7-f94e-4fe7-b9bb-07f391cfcff8)

Still need to - adjust display for larger costs and usages in energy.
                       a case needs to be made to hold the display and the ESP32

**Comment by BottlecapDave** [comment]
This is looking very impressive and glad to see the integration being useful :)

I've done something similar for other house notifications (e.g. vacuum due to run or windows/doors open)

**Reply by anothermort** [reply]
Thanks Dave needs a lot more time spent on it.
But it helps me make the most of your great work.

https://github.com/anothermort/Octopus_mini_HA_Epaper_display
